client/state/jitm/selectors.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/** Internal dependencies */
+import { getSectionName, getSelectedSiteId } from 'state/ui/selectors';
+
+/**
+ * Get the list of available jitms for the current site/section
+ * @param {object} state The application state
+ * @return {array} An array of jitms
+ */
+export const getJITM = ( state ) => get( state, [ 'jitm', 'sitePathJITM', getSectionName( state ) + getSelectedSiteId( state ) ], [] );
+
+/**
+ * Get the top jitm available for the current site/section
+ * @param {object} state Thee application state
+ * @return {object} A jitm
+ */
+export const getTopJITM = ( state ) => {
+	const jitms = getJITM( state );
+
+	if ( jitms.length === 0 ) {
+		return null;
+	}
+
+	return jitms[ 0 ];
```

Comment:
fun fact! we already have an operation whose purpose is to do this…

`lodash.head( [ 1, 2, 3] ) === 1`
`lodash.head( [] ) === undefined`

so it's `undefined` vs `null` but if we need the `null` value we can…

```js
export const getTopJITM = state => head( getJITM( state ) ) || null;
```

Reply:
I always forget about lodash, will update in next pr.